cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -78,16 +79,21 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 			break;
 		}
 
-		long id;
+		long id = 1;
 		synchronized (context) {
 			Long seqValue = (Long) context.get(key);
 			if (seqValue != null) {
 				id = seqValue + 1;
 			}
 			else {
-				id = START_VALUE;
+				try {
+					id = Integer.parseInt(startValue);
+					context.put(key, id);
+				} finally {
+					System.out.println("ERROR with the input value of startValue  " + startValue
```

Comment:
Like this, this will be printed regardless of if there was an error or not.
Also, simply printing to the console like that is not how logging is handled within a transformation.

As I said, rethrowing the exception is probably the easiest way here, e.g.

```
} catch(Exception e) {
  throw new TransformationException("My improved error message", e);
}
```